client/src/pages/Login.tsx

```diff
@@ -1,44 +1,45 @@
 import { observer } from "mobx-react-lite";
 import { authStore } from "../stores/AuthStore";
 import { useState } from "react";
+import Input from "../components/UIElements/Input";
+import Button from "../components/UIElements/Button";
+import { Link, useNavigate } from "react-router-dom";
+import Card from "../components/UIElements/Card";
 
 const Login = observer(() => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const navigate = useNavigate();
 
   const authUser = async () => {
-    await authStore.login(email, password);
-  };
-
-  const handleLogout = () => {
-    authStore.logout();
+    try {
+      await authStore.login(email, password);
+      navigate("/homepage");
+    } catch {
+      alert("incorrect info");
+    }
   };
 
   return (
-    <div>
+    <Card>
```

Comment:
I'm nit sure about our pages being cards 🤔
I feel like a card is a UI Element that should be using inside a page.
What do you think?

Reply:
agree, the purpose of that card comp was just to quickly wrap content and not the whole page. that's why it's narrow.